src/node/snapshotter.h

```diff
@@ -238,7 +248,7 @@ namespace ccf
       last_snapshot_idx = idx;
 
       next_snapshot_indices.clear();
-      next_snapshot_indices.push_back(last_snapshot_idx);
+      next_snapshot_indices.push_back({last_snapshot_idx, false});
     }
 
     bool record_committable(consensus::Index idx)
```

Comment:
The force snapshot flow should also be tested by the snapshotter unit test (`snapshotter.cpp`).

Reply:
Done!